### ISSUE TITLE
Intel TBB not found on Ubuntu 22.04

### ISSUE THREAD

**Issue by Janekdererste** [kit-algo/InertialFlowCutter#13]
Hi,

when I execute `cmake -DCMAKE_BUILD_TYPE=Release ..` I get the following error. 
```
ERROR: Intel TBB NOT found!
-- Looked for Threading Building Blocks in /opt/intel/tbb;/usr/local/include;/usr/include
CMake Error at cmake/FindTBB.cmake:272 (message):
  Could NOT find TBB library.
Call Stack (most recent call first):
  CMakeLists.txt:11 (find_package)
```
However, I have installed the necessary dependencies including `tbb` before
```
$ apt list libtbb-dev libreadline8 libreadline-dev zlib1g-dev
Listing... Done
libreadline-dev/jammy,now 8.1.2-1 amd64 [installed]
libreadline8/jammy,now 8.1.2-1 amd64 [installed]
libtbb-dev/jammy,now 2021.5.0-7ubuntu2 amd64 [installed]
zlib1g-dev/jammy-security,jammy-updates,now 1:1.2.11.dfsg-2ubuntu9.2 amd64 [installed]
```
Is there a workaround to point cmake to the correct lcation of the tbb dependency?

Thanks in advance!

**Comment by larsgottesbueren** [comment]
I just pushed a fix, presumably. Please give it a try and report back. TBB had some backwards-compatibility breaking changes a while back, which somehow affected the FindTBB cmake script as well.

**Comment by Janekdererste** [comment]
Thanks for the quick fix! It works both on ubuntu 20.04 as well as ubuntu 22.04 now.
